src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -262,6 +312,66 @@ export default {
 
             this.time = this.timeFormat.format(datum);
             this.imageUrl = this.imageFormat.format(datum);
+        },
+        getFormatter(key) {
+            let valueFormatter = this.openmct.telemetry.getValueFormatter({
+                format: key
+            });
+
+            return valueFormatter.formatter;
+        },
+        numericImageAge() {
+            let currentTime = this.currentTimeValue();
+            let selectedImage = this.getSelectedImage();
+
+            if (selectedImage === undefined || selectedImage[this.timeKey] === undefined) {
+                return;
+            }
+
+            return currentTime - selectedImage[this.timeKey];
+        },
+        formattedAge() {
+            let age = this.numericImageAge();
+            let result;
+
+            if (!age) {
+                return;
+            }
+
+            result = this.durationFormatter.format(age);
+
+            if (age > EIGHT_HOURS) {
```

Comment:
Wondering if `moment.js`'s `humanize` functions might come in handy here? We already use moment for this sort of thing elsewhere.

Reply:
Good call, using it now for "# hours ago" and "# days ago". @charlesh88 just want to make sure you're cool with this.